client/components/jetpack-logo/index.jsx

```diff
@@ -10,10 +10,14 @@ import PropTypes from 'prop-types';
  * Module constants
  */
 const logoPathSize32 = (
-	<path
-		className="jetpack-logo__icon"
-		d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16c8.8,0,16-7.2,16-16S24.8,0,16,0z M15.2,18.7h-8l8-15.5V18.7z M16.8,28.8 V13.3h8L16.8,28.8z"
-	/> // eslint-disable-line max-len
+	<g>
+		<path
+			className="jetpack-logo__icon-circle"
+			d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z"
+		/>
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="15,19 7,19 15,3 " />
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="17,29 17,13 25,13 " />
```

Comment:
I'm an SVG nerd, sorry 🤓 

We might consider a green circle with white polygons above, or the full green shape (with cut-out) with a white circle below. The `<polygon>`s could be fused into a single path above a simple `circle`, might be the best combination of minimization and clarity.

Also the `<g>` wrapper can be `<React.Fragment>`, we just need a way of working with a single element, we don't actually need an SVG `<g>`. Save an HTML element 🙂 

These aren't blockers, just some notes. Feel free to reach out if you're interested in exploring these options.

Reply:
@sirreal I thought about fusing them in as they were before, but I think this makes them easier to target and understand for styling purposes.